src/cli/managers/config_manager.py

```diff
@@ -17,8 +21,25 @@ def get_application(self, application: str) -> Dict[str, Any]:
     def get_application_from_path(self, path: Path) -> Tuple[str, Dict[str, str]]:
         return "key", {}
 
-    def get_applications(self) -> List[Dict[str, str]]:
-        return [{}]
+    def get_applications(self) -> List[Dict[str, Any]]:
+        """
+        Reads the applications.json config file for getting the local applications
+
+        Returns:
+            A list of applications available in the config file
+        """
+        applications_config = self.__config_dir / 'applications.json'
+        application_list = []
+        if applications_config.is_file():
+            applications = read_json_file(applications_config)
+            for app_name, app_data in applications.items():
+                app_data['name'] = app_name
+                application_list.append(app_data)
+            return application_list
```

Comment:
with an else here you don't have 2 exit paths which is cleaner

Reply:
Added an else statement